src/clusterfuzz/_internal/bot/tasks/utasks/progression_task.py

```diff
@@ -343,28 +455,15 @@ def find_fixed_range(testcase_id, job_type):
   result = _testcase_reproduces_in_revision(testcase, testcase_file_path,
                                             job_type, min_revision)
   if result and not result.is_crash():
-    testcase = data_handler.get_testcase_by_id(testcase_id)
-
-    # Retry once on another bot to confirm our result.
-    if data_handler.is_first_retry_for_task(testcase, reset_after_retry=True):
-      tasks.add_task('progression', testcase_id, job_type)
-      error_message = (
-          'Known crash revision %d did not crash, will retry on another bot to '
-          'confirm result' % known_crash_revision)
-      data_handler.update_testcase_comment(testcase, data_types.TaskState.ERROR,
-                                           error_message)
-      data_handler.update_progression_completion_metadata(
-          testcase, max_revision)
-      return
-
-    data_handler.clear_progression_pending(testcase)
-    error_message = (
-        'Known crash revision %d did not crash' % known_crash_revision)
-    data_handler.update_testcase_comment(testcase, data_types.TaskState.ERROR,
-                                         error_message)
-    task_creation.mark_unreproducible_if_flaky(testcase, True)
-    return
-
+    error_message = ('Known crash revision %d did not crash' % known_crash_revision)
+    testcase.crash_revision = max_revision
+    return uworker_io.UworkerOutput(
+          testcase=testcase,
+          uworker_input=uworker_input,
+          error_message=error_message,
+          error=uworker_msg_pb2.ErrorType.PROGRESSION_NO_CRASH)  
+
+  # TODO: this part is to be moved into a separate task to be scheduled on postprocess.
```

Comment:
nit: Capital T for "this" at the beginning of TODO.
Similar to https://google.github.io/styleguide/pyguide.html#312-todo-comments